baselines/niid_bench/run_all.sh

```diff
@@ -0,0 +1,136 @@
+seed=$1
+
+# iid cifar10
+python -m niid_bench.main_fedavg partitioning=iid dataset_seed=$seed &
+python -m niid_bench.main_scaffold partitioning=iid dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.1 dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid mu=0.001 dataset_seed=$seed &
+python -m niid_bench.main_fedprox partitioning=iid mu=1.0 dataset_seed=$seed &
+python -m niid_bench.main_fednova partitioning=iid dataset_seed=$seed;
```

Comment:
I'm not sure if it's ideal to run experiment in parallel like this, but if your machine is fine with it, then that's 💯

Reply:
I agree and my machine can handle this. I have provided this file as a reference and it can be changed depending on the machine specifications.